### ISSUE TITLE
Sorry to bother you, have you trained D1 or D2? ? ?

### ISSUE THREAD

**Issue by limengfei3675** [signatrix/efficientdet#40]
Sorry to bother you, have you trained D1 or D2? ? ?

**Comment by CraigWang1** [comment]
I have the same question too

**Comment by jinfagang** [comment]
There were no one replicated D1 above efficientdet performance in community.....

**Comment by zylo117** [comment]
> There were no one replicated D1 above efficientdet performance in community.....

yes, i think the missing activation and the bn momentum is the main issue here, so I re-implement efficientdet based on this repo.

the performance seems nice.
[Yet-Another-EfficientDet-Pytorch](https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch)
